docs/flutter-core/quickstart.mdx

```diff
@@ -69,6 +69,27 @@ You'll learn how to:
 
 After importing the package, perform the following steps for Android and iOS.
 
+### For Android
+
+Set `compileSdkVersion 33` and `minSdkVersion 21` inside `build.gradle` file at the `<project root>/android/app/build.gradle` file.
+
+```kotlin
+  defaultConfig {
+        ...
+
+        compileSdkVersion 33
+        minSdkVersion 21
+
+        ...
+    }
+```
```

Comment:
(suggestion): Setting `compileSdkVersion` and especially `minSdkVersion` should be client's call. 
We can inform clients that our SDK supports 
- minSdkVersion `21` or higher & 
- compileSdkVersion `33` or higher